code/game/objects/items/stacks/stack.dm

```diff
@@ -422,7 +422,7 @@
 /obj/item/stack/proc/can_merge(obj/item/stack/check)
 	if(!istype(check, merge_type))
 		return FALSE
-	if(mats_per_unit != check.mats_per_unit)
+	if((mats_per_unit != check.mats_per_unit) && is_cyborg)
```

Comment:
is_cyborg returns 2 lines later though

Reply:
so we swap this IF and the one below? Didn't knew RETURN ends proc